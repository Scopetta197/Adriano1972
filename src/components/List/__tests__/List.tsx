import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import List from "../index";

describe("<List />", () => {
  it("renders", () => {
    const tree = render(
      <MemoryRouter>
        <List
          items={[
            {
              id: 1,
              name: "Welly",
              phone: "+886 0970...",
              email: "hivoid19@gmail.com",
              website: "https://wellyshen.com",
            },
          ]}
        />
      </MemoryRouter>
    ).container.firstChild;

    expect(tree).toMatchSnapshot();
  });
});
