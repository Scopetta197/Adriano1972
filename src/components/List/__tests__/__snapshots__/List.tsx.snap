// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> renders 1`] = `
<div
  class="UserList"
>
  <h4>
    User List
  </h4>
  <ul>
    <li>
      <a
        href="/UserInfo/1"
      >
        Welly
      </a>
    </li>
  </ul>
</div>
`;
